src/components/bouquets/BouquetDatasetCard.vue

```diff
@@ -1,56 +1,35 @@
 <script setup lang="ts">
 import { DatasetCard } from '@datagouv/components'
-import { type DatasetV2 } from '@datagouv/components'
-import { ref, watch, toRef, type Ref } from 'vue'
+import type { Dataset, DatasetV2 } from '@datagouv/components'
```

Comment:
```suggestion
import type { DatasetV2 } from '@datagouv/components'
```

PI mon vscode affiche ça. Je n'ai rien configuré de particulier et les deux extensions sont dans les recommendations du dépôt il me semble.

<img width="684" alt="Capture d’écran 2024-09-12 à 08 09 11" src="https://github.com/user-attachments/assets/ee498608-5aa6-4548-b71f-c087c4b9a91e">

Etonnamment un `npx eslint src/` ne l'attrape pas...